edge/pkg/edged/edged_status.go

```diff
@@ -348,6 +351,11 @@ func (e *edged) setMemInfo(total, allocated v1.ResourceList) error {
 	mem := resource.MustParse(strconv.FormatInt(totalMem, 10) + "Mi")
 	total[v1.ResourceMemory] = mem.DeepCopy()
 
+	reservedMem := constants.DefaultSystemReservedMEM
+	if config.Config.SystemReserved["memory"] != "" {
+		reservedMem = config.Config.SystemReserved["memory"]
+	}
+	reservationMemory := resource.MustParse(reservedMem)
```

Comment:
Avoid redundant variables
```suggestion
	var  reservationMemory resource.Quantity
	if config.Config.SystemReserved["memory"] != "" {
		reservationMemory = resource.MustParse(config.Config.SystemReserved["memory"])
	}else{
		reservationMemory = resource.MustParse(constants.DefaultSystemReservedMEM)
	}
```